store/modules/actions/payment.js

```diff
@@ -51,7 +50,6 @@ export async function sendCosmosPayment(
     commit(types.UI_SET_HIDE_TX_DIALOG_ACTION, !showDialogAction);
     commit(types.PAYMENT_SET_PENDING_HASH, txHash);
     commit(types.PAYMENT_SET_PENDING_TX_INFO, { from, to, value });
-    if (isWait) await included();
```

Comment:
What if we want to skip waiting for tx confirm (`isWait: false`)?  Does ` await transferCosmos` wait for tx confirm? Can we not wait for confirm according to `isWait` ?

Reply:
true. Will add included back.